static/app/utils/theme.tsx

```diff
@@ -544,6 +544,9 @@ const commonTheme = {
 
   space: [0, 8, 16, 20, 30],
 
+  // used as a gradient,
+  businessIconColors: ['#EA5BC2', '#6148CE'],
```

Comment:
Do we have colors that are "close" to these?

Reply:
@EvanPurkhiser purple100 -> purple300 looks good to me:

![Screen Shot 2021-05-06 at 3 01 13 PM](https://user-images.githubusercontent.com/8533851/117370955-fcc65980-ae7b-11eb-83a0-f90dd767f16d.png)

Keep?

Reply:
yep